vendor_skins/Evercast/CI/before-script-osx.sh

```diff
@@ -3,14 +3,16 @@ export PATH=/usr/local/opt/ccache/libexec:$PATH
 mkdir build
 cd build
 cmake \
--DDepsPath=/tmp/obsdeps \
+-DENABLE_SCRIPTING=OFF \
+-DDepsPath=~/obsdeps \
 -DCMAKE_BUILD_TYPE=RELEASE \
 -DCMAKE_INSTALL_PREFIX=/opt/ebs \
--DVLCPath=$PWD/../../vlc-master \
--DQTDIR=/usr/local/Cellar/qt/5.10.1 \
--DCMAKE_OSX_DEPLOYMENT_TARGET=10.10 \
+-DVLCPath=~/vlc-3.0.4 \
+-DQTDIR=/usr/local/Cellar/qt/5.14.1 \
+-DCMAKE_OSX_DEPLOYMENT_TARGET=10.12 \
 -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl@1.1 \
--Dlibwebrtc_DIR=/tmp/libWebRTC-73.0-x64-Rel-COMMUNITY-BETA/cmake \
--DOBS_VERSION_OVERRIDE=23.2.0 \
+-Dlibwebrtc_DIR=~/libwebrtc/cmake \
+-DBUILD_BROWSER=false \
+-DOBS_WEBRTC_VENDOR_NAME=Evercast \
+-DOBS_VERSION_OVERRIDE=2.5.0 \
```

Comment:
There's a version constant for OBS built into a .h file in the project.  This gets used when displaying the version, e.g. in the title bar of the main window.  It's a version *override* because the default version (23.2.0) will get applied if you don't set this value.